Helpers/InvestigationsHelpers.cs

```diff
@@ -31,7 +35,10 @@ await LykosAvatarMethods.UserOrMemberAvatarURL(infringingMessage.Author, infring
                 embed.AddField(extraField.name, extraField.value, extraField.inline);
 
             if (content == default)
-                content = $"{Program.cfgjson.Emoji.Denied} Deleted infringing message by {infringingMessage.Author.Mention} in {infringingMessage.Channel.Mention}:";
+                if (wasAutoModBlock)
+                    content = $"{Program.cfgjson.Emoji.Denied} Detected infringing message (blocked by AutoMod) by {infringingMessage.Author.Mention} in {infringingMessage.Channel.Mention}:";
```

Comment:
This feels too wordy
```suggestion
                    content = $"{Program.cfgjson.Emoji.Denied} Detected infringing AutoMod message by {infringingMessage.Author.Mention} in {infringingMessage.Channel.Mention}:";
```

Reply:
I agree but wasn't really sure how to put it. This works. Thanks!